Makefile

```diff
@@ -46,6 +46,6 @@ build:
 	cd src && make build
 
 clean:
-	cd src && make clean
+	[ -r src/Makefile ] && ( cd src && make clean ) || true
```

Comment:
can't you just `-cd src && make clean` (see https://www.gnu.org/software/make/manual/make.html#Errors)